store/mocks/store.go

```diff
@@ -0,0 +1,103 @@
+// Code generated by mockery v1.0.0. DO NOT EDIT.
+
+package mocks
```

Comment:
Can this package be called "mock"?

Reply:
hm, does some other projects uses mock and not mocks?
by the way:
```
 -outpkg string
       name of generated package (default "mocks")
 -output string
       directory to write mocks to (default "./mocks")
```
so this is easy to be setted to mock

Reply:
@asim @vtolstov 
Updated.